### PR TITLE
Release preparations

### PR DESCRIPTION
**Pull request by favreau**

### CODE REVIEW

**Comment by favreau** [comment]
Retest this please

**Comment by favreau** [comment]
Retest this please

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[submodule "CMake/common"]
+	path = CMake/common
+	url = https://github.com/Eyescale/CMake
+        branch = master
+        shallow = true
```
Comment (rdumusc) [inline .gitmodules]: no branch and shallow

File: CMakeLists.txt [hunk 2]
```diff
@@ -10,19 +10,23 @@ project(Brayns VERSION 0.3.0)
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake
                               ${CMAKE_SOURCE_DIR}/CMake/common)
+if(NOT EXISTS ${CMAKE_SOURCE_DIR}/CMake/common/Common.cmake)
+  message(FATAL_ERROR "CMake/common missing, run: git submodule update --init")
+endif()
+
 include(GitExternal)
```
Comment (rdumusc) [inline CMakeLists.txt]: remove  include(GitExternal)

File: README.md [hunk 3]
```diff
@@ -95,7 +95,7 @@ Clone OSPRay 1.2.0 in the same folder level as ISPC compiler
   git clone https://github.com/BlueBrain/Brayns.git
```
Comment (rdumusc) [inline README.md]: git clone --recursive https://github.com/BlueBrain/Brayns.git

**Pull request merged**